File: JavaScritpt/Arrays/Frequency.js
function Friquency(arr,num,n){
    var i,count=0;
    for(i=0;i<n;i++){
        if(arr[i]==num){
            count=count+1;
        }
    }
    return count;
}
var arr=[1,1,1,1,1,2,2]
var n=arr.length;
console.log(Friquency(arr,2,n));

File: C++Code/Arrays/Frequency.cpp
#include<iostream>
using namespace std;
void findFrequency(int arr[], int n)
{
    int i;
    for (i = 0; i < n; i++)
    {  
        int count = 1;
        int tem = arr[i];
        for (int j = i + 1; j < n; j++)
        {
            if (tem == arr[j])
            {
                count = count + 1;
                i++;
            }
        }
        cout << tem << "->";
        cout << count << endl;
    }
}
int main()
{
    int arr[] = {8, 9, 100, 200, 200, 200, 200};
    int n = sizeof(arr) / sizeof(arr[0]);
    findFrequency(arr, n);
}